react/Field/index.spec.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import Field from './'
+import { shallow } from 'enzyme'
+
+describe('Field component', () => {
+  afterEach(() => {
+    jest.resetAllMocks()
+  })
+
+  it('should expect object type value for type=select', () => {
+    // mock for proptypes
+    console.error = jest.fn(message => {
+      throw new Error(message)
+    })
```

Comment:
Prefer to replace console.error in a beforeEach (since it is repeated twice) with a spyOn (to be restored in a afterEach), since we are modifying a global object, it is better to be hygienic even if I think Jest resets the jsdom environment between each test.

Reply:
This test file could have more tests about the component itself since we just need a simple mock for those two test case and that could break other test case I prefer to keep them here